source/PlanetPanel.cpp

```diff
@@ -105,6 +106,26 @@ void PlanetPanel::Step()
 		return;
 	}
 
+	// Determine which shops are conditionally available.
+	// This needs to wait until the first Step call instead of being
+	// done in the constructor because the constructor is created
+	// before all of the player's landing logic is completed, which
+	// can cause certain conditions to return unexpected results.
```

Comment:
This is okay for this PR, if we add a statement to the wiki that condition-changes made while being landed on the planet are not guaranteed to have effect until the next landing.

For the longer term future I hope that we will get to a situation where the conditions are updated in realtime whenever we have an outfitter screen open. (Allowing the purchase of one outfit to immediately affect the stock of other outfits.)

Reply:
Yeah, I would like to have this be more reactive in the future. I'll probably add a TODO comment mentioning to update this when condition checking is made more performant.

Reply:
```suggestion
	// can cause certain conditions to return unexpected results.
	// TODO: Determine stock on the fly after condition entries can be subscribed to.
```